.github/copilot-instructions.md

```diff
@@ -0,0 +1,29 @@
+# TypeScript Project Coding Standards
+
+You are an expert TypeScript developer. When generating code for this project, adhere to the following naming conventions and architectural rules.
+
+## General Guidelines
+- **Respond Specifically to Prompts**: Never do things not explicitly requested.
+
+## Variable Naming Rules
+- **No Single-Letter Variables:** Never use single-letter variable names (e.g., `i`, `e`, `v`, `k`, `v`).
+- **Descriptive Naming:** Use clear, descriptive names that convey the intent and data type of the variable.
+- **Loop Iterators:** Instead of `i`, use `index`. Instead of `e`, use a specific name like `user` or `account`.
+- **Error Handling:** Use `error` instead of `e` in catch blocks.
+
+## TypeScript Best Practices
+- **Strict Typing:** Avoid the `any` type if at all possible. Use `unknown` if the type is truly dynamic, or define proper `Interfaces` and `Types`.
+- **Functional Approach:** Prefer `map`, `filter`, and `reduce` over traditional `for` loops where readability is maintained.
+- **Explicit Returns:** Always define explicit return types for functions to ensure type safety across the boundary.
+
+## Examples
+
+### Incorrect
+```typescript
+const data = list.map(x => x.id);
+
+try {
+  // ...
+} catch (e) {
+  console.error(e);
+}
```

Comment:
Should there be a "Correct" example as well?  Or is that just extra useless tokens for the AI to process?